### DISCUSSION TITLE
Where can I find the log files for the installed system?

### DISCUSSION THREAD

**Discussion by shawty** [modoboa/modoboa#3160] [Help]
Install went ok, no issues, nothing missing...

![image](https://github.com/modoboa/modoboa/assets/1876454/b0b40427-eb58-4f0b-863a-6411349e9dde)

But no matter how many times I try to install this, as soon as I try to open the web-admin...

login screen comes up fine

![image](https://github.com/modoboa/modoboa/assets/1876454/339d02b4-4474-4e42-b43b-0da88bd54b52)

immediately followed by this

![image](https://github.com/modoboa/modoboa/assets/1876454/bc128014-78db-483e-a6ae-3a64bb367bdb)

I've started from scratch several times now, each time on a brand new Ubuntu 20.04 LTS install (Running in a VM if it matters) 30gb disk space and as much memory as it needs (Dynamically allocated) up to 16gb in total.

I've looked in all the normal places for log files, nothing looks out of place.

Looked in the "web logs" in /var/log/nginx, and again nothing at all out of place, the ONLY thing I see in the web logs is that the post request for login returned a 500 error (Which I expect would be the case given the message) , are there any other log files I can check?

Bear in mind, I CANNOT get ANY of the GUI's up in the browser.  even using   "/new-admin" asks me to log in, then just returns a message similar to the one above.

Cheers
Shawty

**Comment by tonioo** [comment]
Might be caused by this issue: https://github.com/modoboa/modoboa/issues/3154.
A 2.2.4 version will be released soon

**Reply by shawty** [reply]
thanks again @tonioo , shall I just wait for 2.2.4 to be released then and try again after that?

I can hold off installing the new mail server for a little while as long as it's not too long.

cheers
shawty

**Reply by tonioo** [reply]
2.2.4 has been released yesterday

**Reply by shawty** [reply]
so if I do an install right now, on a fresh machine, using the installer, I will now get the newer version?

If yes, then I'll test it later today.

**Reply by tonioo** [reply]
if it's a complete new install then yes

**Reply by shawty** [reply]
great stuff, I'll update here what happens.
